resources/language/resource.language.sv_se/strings.po

```diff
@@ -198,16 +198,16 @@ msgstr "Aktivera DEMO-läge"
 
 msgctxt "#30805"
 msgid "Show an UpNext pop-up…"
-msgstr "Visa en popup för UpNext…"
+msgstr "Visa en UpNext-popup…"
 
 msgctxt "#30807"
 msgid "Show an UpNextSimple pop-up…"
-msgstr "Visa en enkel popup för UpNext…"
+msgstr "Visa en enkel UpNext-popup…"
```

Comment:
Any reason this is `Visa en enkel UpNext-popup…` and not `Visa en enkel UpNext popup…` or `Visa en enkel UpNext pop-up…`?